package.json

```diff
@@ -68,6 +69,7 @@
     "eslint": "7.21.0",
     "eslint-formatter-pretty": "4.0.0",
     "eslint-import-resolver-webpack": "0.13.0",
+    "gatsby": "3.0.4",
```

Comment:
Vercel's framework detection looks into package.json and wants to see gatsby there.  It's a redundant entry here because it actually belongs into the workspace folders only.  But it is not bloating beause we automate using the same version everywhere (renovate bot) so it's effectively symbolic.